camunda-formio-tasklist-vue/src/components/search/TaskListSearch.vue

```diff
@@ -279,11 +285,14 @@ export default class TaskListSearch extends Vue {
     }
     Vue.set(this.selectedSearchQueries, index, searchitem);
     this.operator[index] = this.selectedSearchQueries[index].compares[0];
-    if(this.selectedSearchQueries[index].type==='date'){
```

Comment:
does compares array has at least one element?- will it throw any error?

Reply:
Compares array will have all it's elements prefilled every time. Hence it won't throw any error during program execution

![image](https://user-images.githubusercontent.com/70306694/118595249-5c036280-b7c8-11eb-8946-015f84422005.png)